rust/fastsim-cli/src/bin/fastsim-cli.rs

```diff
@@ -226,13 +227,14 @@ pub fn main() -> anyhow::Result<()> {
         RustCycle::from_file(adopt_hd_string)
     } else {
         //TODO? use pathbuff to string, for robustness
```

Comment:
@kylecarow , is this comment deprecated here?

Reply:
Not sure, that's leftover from Andrew Guttman. I think we could delete the TODO.